### ISSUE TITLE
Question: Where is this Linux icon from?

### ISSUE THREAD

**Issue by Alex-Sour** [DAGINATSUKO/www-rpcs3#82]
## Introduction

The icon at [/img/icons/buttons/linux.png](https://rpcs3.net/img/icons/buttons/linux.png) looks amazing, and I wanted to use it on my website.

## My efforts

I looked in the [branding page](https://rpcs3.net/branding) and found the list of icon packs used, then proceeded to find them (the packs aren't linked, only the authors, and Google search rarely reveals the pack itself) and look through every icon in them to try to find that Linux icon, and got nothing.

I've searched Flaticon and Freepik enough (for both "Linux" and "Penguin" that I've come to the conclusion that this icon is not on those platforms.

My current theory is that this icon was created by you (DAGINATSUKO), but I have decided to ask here as my last resort.

## Why

This icon looks amazing. Unlike every other Linux icon I've ever seen, it goes for a more minimal approach and really gets a nice, modern feel while still communicating the classic penguin.

I've looked everywhere, and no other penguin or Linux icon seems to have a design like this one.

## Conclusion

In conclusion, I don't know why I wrote so much stuff, this would've probably been sufficient:

> Hey where is [/img/icons/buttons/linux.png](https://rpcs3.net/img/icons/buttons/linux.png) from? I want to use it on my site

But I do like writing, so please forgive my long issue.

I hope that you could tell me where you got this icon, and hopefully it is under a license that would allow me to use it on my site.

Thanks,
Alex_Sour

**Comment by Alex-Sour** [comment]
Is there anyone with Information

**Comment by Alex-Sour** [comment]
Thank you. Your response has been recorded.

**Comment by Alex-Sour** [comment]
Actually, do you know what the license is or are you just bravely ignoring copyright?

Thank you.

**Comment by Alex-Sour** [comment]
I have vectorized the image (vectorized)
![Linux Modern](https://user-images.githubusercontent.com/52638772/168888198-c46635a8-3c01-438e-a64a-c1b5caf71c1e.svg)

SVG
```svg
<svg version="1.2" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 252 252" width="252" height="252"><style></style><path fill-rule="evenodd" d="m140.4 0.8c0.8 0.1 2.8 0.4 4.6 0.6 48.1 6.5 89.7 43.7 102.5 91.4 4.9 18.3 5.8 41 2.1 57-2.5 11-4.2 16.5-7.1 23.6-4.1 10.1-11.9 23.3-17.7 30.2-0.5 0.7-1.7 2.2-2.6 3.2-9.5 11.9-25.9 24.6-41.4 32.1-6.7 3.3-6.4 3.3-5.8 0 3.4-16.6 3.8-33.4 1.2-44.5-2-7.9-5.9-19.3-9.4-26.8-6.7-14.4-2.8-31.9 8.8-39.8 9.8-6.7 16.9-8.1 44.4-8.7 15.3-0.4 16.1-0.6 16.3-3.6 0.1-1.5 0-1.7-1.5-3-2.5-2.1-4.7-3.2-13.5-6.5-1.5-0.6-3.1-1.2-3.5-1.4-0.4-0.2-3.2-1.3-6.1-2.4-12.4-4.7-16.8-7.6-23.9-15.7-1.8-2.1-4.1-5.2-6.4-8.7-14.3-21.6-29.8-31.1-55-33.8-19.5-2.1-44.2 5.4-62.4 19.1-21.1 15.8-40.5 37.7-58.5 66.3-1 1.6-2.5 4-3.2 5.2l-1.4 2.1-0.3-5.1c-1.5-26.6 6.4-54.9 21.7-77 22.2-32 57.1-51.9 95.7-54.4 7.4-0.5 16.9-0.2 22.4 0.6zm-51.8 79.5c-19.6 21.8-16.7 39.7 10 60.5 30 23.4 44.1 42.5 50.2 68 2.7 11.6 2.7 28.6-0.1 37.8l-1 3-1.7 0.3c-5.9 0.9-9.2 1.2-17.8 1.3-22.6 0.3-40.9-3.9-60.2-13.9-4.2-2.1-5.2-2.7-10.6-6.2-21.3-13.7-41.3-38.4-48.8-60.3l-1.1-3.2 4-7.8c6.2-12 15.6-29 20-35.9 13.3-21.3 22.3-32.5 33-41.5 6.2-5.3 16-11.2 21.8-13.2 10.2-3.6 11.2 1.1 2.3 11.1zm69.8 2c0 5.2-10.8 7-14.8 2.5-2.3-2.6-1.2-5.2 2.8-6.6 5.2-1.9 12 0.5 12 4.1z"/></svg>
```

**Comment by NayamAmarshe** [comment]
@Alex-Sour Looks amazing! What a nice logo.

**Comment by DAGINATSUKO** [comment]
> 



> Actually, do you know what the license is or are you just bravely ignoring copyright?
> 
> Thank you.



> Actually, do you know what the license is or are you just bravely ignoring copyright?
> 
> Thank you.

I'm actually not sure, technically I had to remake the logo because the one on their website is very low res so I had to re-draw it and then vectorize it. Other than that, it has a black version, white version and a color gradient version on the website.

**Comment by NayamAmarshe** [comment]
@DAGINATSUKO https://www.linux.org/threads/linux-org-logo-licensing.40445/

**Comment by Alex-Sour** [comment]
Oh no, that's awful!
